sites/public/src/components/shared/MapControl.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react"
```

Comment:
To get this to build I had to change this line to `import React, { RefObject } from "react"`